src/main/java/wolox/training/models/User.java

```diff
@@ -56,6 +56,9 @@
     public User() {
         this.books = new ArrayList<>();
     }
+    public User(String name) {
```

Comment:
This change was in another branch, you are going to have problems in the merge

Reply:
Please review again, this branch wasn't update with main.